t.cs

```diff
@@ -0,0 +1,23 @@
+public class Test
+{
+	public bool T(bool c)
+	{
+        if (c)
+		{
+			T(false);
+		}
+		return true;
+	}
+
+    public event EventHandler SomeEvent
+    {
+        add
+        {
+            // add event subscriber here
+        }
+        remove
```

Comment:
Elements should be separated by blank line.

```suggestion

        remove
```